nixos/modules/epics-environment.nix

```
@@ -51,12 +51,63 @@ in
       type = with lib.types; listOf str;
       default = [ ];
     };
+    openCAFirewall = lib.mkOption {
+      description = ''
+        This option enable the default ports of the Channel Access protocol.
+        If in your IOC you manually change the ports.
+        You will need to open them manually.
+        With the NixOS option {nix:option}`networking.firewall.allowedTCPPorts` & {nix:option}`networking.firewall.allowedUDPPorts`
+      '';
+      type = lib.types.bool;
+      default = false;
+
+    };
+    openPVAFirewall = lib.mkOption {
+      description = ''
+        This option enable the default ports of the pvAccess protocol.
+        If in your IOC you manually change the ports.
+        You will need to open them manually.
+        With the NixOS option {nix:option}`networking.firewall.allowedTCPPorts` & {nix:option}`networking.firewall.allowedUDPPorts`
```

Note: you might have to add the paragraph about the PVA broadcast discovery.

```suggestion
        Open the default ports of the pvAccess protocol.

        :::{note}
        If you changed the pvAccess ports in your IOC,
        open these ports in the firewall manually
        by using `networking.firewall.allowedTCPPorts`
        and `networking.firewall.allowedUDPPorts`.
        :::
```